src/src/com/microsoft/aad/adal/StorageHelper.java

```diff
@@ -466,7 +466,7 @@ final synchronized private SecretKey getSecretKeyFromAndroidKeyStore() throws IO
             final byte[] encryptedKey = readKeyData(keyFile);
             sSecretKeyFromAndroidKeyStore = unwrap(wrapCipher, encryptedKey);
             Logger.v(TAG, "Finished reading SecretKey");
-        } catch (Exception ex) {
+        } catch (GeneralSecurityException ex) {
```

Comment:
readKeyData would throw out the IOException, I think we should also catch the exception here.